src/EnergyPlus/DuctLoss.cc

```diff
@@ -731,6 +735,19 @@ namespace DuctLoss {
                     errorsFound = true;
                 }
             }
+            // Get an index of splitter and mixer for a single AirLoop
+            for (int SupAirPath = 1; SupAirPath <= state.dataZoneEquip->NumSupplyAirPaths; ++SupAirPath) {
+                for (int CompNum = 1; CompNum <= state.dataZoneEquip->SupplyAirPath(SupAirPath).NumOfComponents; ++CompNum) {
+                    state.dataDuctLoss->SplitterNum = state.dataZoneEquip->SupplyAirPath(SupAirPath).SplitterIndex(CompNum);
+                    break;
+                }
+            }
```

Comment:
I do not understand these loops. Shouldn't there be some sort of `if` test?

Reply:
@mjwitte Here are my answers based on your question order:

1. A single airloop is checked in AirflowNetwork. I can add an error check for a single airloop.
2. The present model assumes a single splitter and mixer for a single airloop, Any other choices will be expanded later.
3. No for the present model. The airflownetwork model does not allow multiple mixers and splitters for a single airloop. More choices will be expanded later.
4. Not yet for the present model.
5. I am going to add more error checks to allow a single splitter and mixer for a single airloop.

Reply:
@mjwitte You are right. 
I remove state.dataZoneEquip->SupplyAirPath and state.dataZoneEquip->ReturnAirPath and set both variables of SplitterNum and MixerNumto 1,
Thanks.